README.md

```diff
@@ -1,31 +1,56 @@
-|     Backend CI | [![Build status](https://github.com/ibis-project/ibis/actions/workflows/ibis-backends.yml/badge.svg)](https://github.com/ibis-project/ibis/actions/workflows/ibis-backends.yml?query=branch%3Amaster) |
-|       Coverage | [![Codecov branch](https://img.shields.io/codecov/c/github/ibis-project/ibis/master.svg)](https://codecov.io/gh/ibis-project/ibis)                                                                    |
+[![Documentation Status](https://img.shields.io/badge/docs-docs.ibis--project.org-blue.svg)](http://ibis-project.org)
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/ibis-framework/badges/version.svg)](https://anaconda.org/conda-forge/ibis-framework)
+[![PyPI](https://img.shields.io/pypi/v/ibis-framework.svg)](https://pypi.org/project/ibis-framework)
+[![Build status](https://github.com/ibis-project/ibis/actions/workflows/ibis-main.yml/badge.svg)](https://github.com/ibis-project/ibis/actions/workflows/ibis-main.yml?query=branch%3Amaster)
+[![Build status](https://github.com/ibis-project/ibis/actions/workflows/ibis-backends.yml/badge.svg)](https://github.com/ibis-project/ibis/actions/workflows/ibis-backends.yml?query=branch%3Amaster)
+[![Codecov branch](https://img.shields.io/codecov/c/github/ibis-project/ibis/master.svg)](https://codecov.io/gh/ibis-project/ibis)
 
-Ibis is a Python library to help you write expressive analytics at any scale,
-small to large. Its goal is to simplify analytical workflows and make you more
-productive.
+## What is Ibis?
 
-Install Ibis from PyPI with:
+Ibis is a Python library that provides a lightweight, universal interface for data wrangling. It helps Python users explore and transform data of any size, stored anywhere.
 
-```sh
-pip install ibis-framework
-```
+Ibis has three primary components:
 
-or from conda-forge with
+1. **A dataframe API for Python**.
+   This means that Python users can write Ibis code to manipulate tabular data.
+2. **Interfaces to 10+ query engines.**
+   This means that wherever data is stored, data scientists can use Ibis as their API of choice to communicate with any of those query engines.
+3. **Deferred execution**.
+   Ibis uses deferred execution, meaning that execution of code is pushed to the query engine.
+   This means users can execute at the speed of their backend, not their local computer.
 
-```sh
-conda install ibis-framework -c conda-forge
-```
+## Why Use Ibis?
+
+Ibis aims to be a future-proof solution to interacting with data using Python and can accomplish this goal through its main features:
+
+- **Familiar API**: Ibis’s API design borrows from popular APIs like pandas and dplyr that most users already know and like to use.
+- **Consistent syntax**: After learning how to use Ibis, users can use the same API and concepts no matter where their data lives.
+  **Learn once, write anywhere**.
```

Comment:
I guess we're going with this. I'm growing more fond of this but I still think "learn" is a bit passive. I'd like to push us to think about something more aggressive without alienating users. Maybe:
*A Universal Python API
*Standardized Python Query Language
*Python made Portable
Again, feel free to over rule me, but learn in my head highlights the lack of guarantees, when in reality there are clusters of functionality that are very portable, and those clusters needs to be articulated better.

Reply:
How about `Portable Python Analytics`?

Reply:
Yeah I'm not a big fan of "learn" if I'm being honest; if we went with "Learn once, write anywhere" as a bit then the first word someone runs into wrt Ibis will be "learn", which sounds like a homework assignment.

Learning isn't a bad thing obviously, and it's something a new user will be doing anyways, but it, to me, feels like an unwelcoming hurdle to Ibis vs demonstrating that Ibis is something you can jump into and immediately get long-term value out of _while_ learning a universal interface to tabular data.

For this particular bullet, does this make sense?

<hr>

- **Consistent syntax**: After learning how to use Ibis, users can use the same API and concepts no matter where their data lives. **Ibis aims to be universal python API for tabular data, big or small.**.

<hr>

Or do we think there's a nuance that this is eliding?

Reply:
I think you can probably just keep the last sentence.

Reply:
@datametrician said:
> Again, feel free to over rule me, but learn in my head highlights the lack of guarantees, when in reality there are clusters of functionality that are very portable, and those clusters could be articulated better.

We're not necessarily making a long-term decision, just incremental progress to push the ball forward.  It's too easy to get caught up in all these details when reviewing a PR; the ideal is to take this PR and then whenever you/we/whomever has some better words, they can submit a new PR and we'll take that one too.  Even if it's tomorrow :)

Reply:
Noted @saulpw and understood.

I love Portable Python Analytics @cpcloud!